spec/models/term_spec.rb

```diff
@@ -14,19 +14,42 @@
       expect(term.uuid).not_to be blank?
     end
 
-    it 'return error when uri invalid' do
-      expect {
-        FactoryBot.create(:external_term, uri: 'ldfkja')
-      }.to raise_error ActiveRecord::RecordInvalid
+    context 'with an invalid URI' do
+      let(:term) { FactoryBot.build(:external_term, uri: 'ldfkja') }
+
+      it 'returns validation error' do
+        expect(term.save).to be false
+        expect(term.errors.full_messages).to include 'Uri is invalid'
+      end
     end
 
-    it 'returns error if uri is missing' do
-      expect { FactoryBot.create(:external_term, uri: '') }.to raise_error ActiveRecord::RecordInvalid
+    context 'with a missing uri' do
+      let(:term) { FactoryBot.build(:external_term, uri: '') }
+
+      it 'returns validation error' do
+        expect(term.save).to be false
+        expect(term.errors.full_messages).to include 'Uri can\'t be blank'
+      end
     end
 
-    it 'returns error if uri is already represented in vocabulary' do
-      term
-      expect { FactoryBot.create(:external_term) }.to raise_error ActiveRecord::RecordInvalid
+    context 'when uri is already represented in vocabulary' do
+      let(:term_0) { FactoryBot.create(:external_term) }
+      let(:term)   { FactoryBot.build(:external_term, vocabulary: term_0.vocabulary) }
+
+      it 'returns validation error' do
+        expect(term.save).to be false
+        expect(term.errors.full_messages).to include 'Uri has already been taken'
```

Comment:
```suggestion
        expect(term.errors.full_messages).to include 'Uri has already been added to this vocabulary'
```
Are you okay with my suggested error wording change?

Reply:
The errors are default rails wording, but I think it is possible to change them.

Reply:
Oh okay.  It's not a huge deal to leave as is.  I just thought it would be slightly clearer to the client if possible to change.

Reply:
I was able to customize this!

Reply:
Awesome!